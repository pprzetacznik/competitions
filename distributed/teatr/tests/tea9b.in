100000000 2 40 -1 1000000 2048415744931607417
