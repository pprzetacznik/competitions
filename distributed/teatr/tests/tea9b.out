3299985374588428
